sys/mips/include/param.h

```diff
@@ -184,13 +188,27 @@
  * For a large kernel stack page the KSTACK_SIZE needs to be a page size
  * supported by the hardware (e.g. 16K).
  */
+#ifdef __CHERI_PURE_CAPABILITY__
+/*
+ * Cheri-128 uses two 16K pages for the kernel stack, without any guard pages.
```

Comment:
```suggestion
 * CHERI-128 uses two 16K pages for the kernel stack, without any guard pages.
```

Reply:
(But also maybe we should just drop the `-128`, we have no 256 configs any more. Plus this probably wants to say `purecap`/`pure-capability` somewhere as it doesn't apply to a hybrid CHERI kernel.)